### DISCUSSION TITLE
Using `getStaticPaths` in combination with `trailingSlash: true` leads to invalid json prefetching

### DISCUSSION THREAD

**Discussion by bernardodestefano** [vercel/next.js#18014] [Help]
Using getStaticPaths in combination with trailingSlash: true leads to invalid json prefetching. For a link to a dynamic route, the json file that is prefetched includes an invalid slash in the path, causing a 404 to be returned.

Example:
While on the homepage, clicking on the navigation for the /about page request a resource "_next/data/it/about/.json" instead of "_next/data/it/about.json"

anyone else experiencing this?  😢

**Comment by emadicio** [comment]
https://github.com/vercel/next.js/issues/17813 Leaving related issue here for ref.
